SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005847	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire continuously in a straight line in front of you until the maximum range of fire. The attacked enemies might be pushed back.
+SKILL_20151224_005848	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Use the butt of the gun to attack an enemy in close distance.
+SKILL_20151224_005849	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Stab an enemy by using the bayonet equipped to the rifle. The stabbed enemy might be pushed back.
+SKILL_20151224_005850	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionTime}#%{nl}Recover 5% of Maximum HP{nl}Additional HP recovery: #{CaptionRatio}# ~ #{CaptionRatio2}#{nl}Magic Circle duration: 40 second
+SKILL_20151224_005851	Application count: #{CaptionRatio}#{nl}Duration: 60 second
+SKILL_20151224_005852	HP Recovery increase: #{CaptionRatio}#{nl}HP Recovery time reduction: #{CaptionRatio2}#second{nl}Torch duration: #{CaptionTime}#second
+SKILL_20151224_005853	Increase the count of effect for you and your party members.
+SKILL_20151224_005854	Maximum count increase: #{CaptionRatio}#{nl}Duration: #{CaptionTime}#second
+SKILL_20151224_005855	Lure the zombies by using the skull of a monkey and attack enemies for a period of time.
+SKILL_20151224_005856	Strength increase: #{CaptionRatio}#{nl}Duration: #{CaptionRatio2}#second{nl}Glyph duration: 15 second
+SKILL_20151224_005857	Maximum capsule count: #{CaptionRatio}#
+SKILL_20151224_005858	Attack: 250% + #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%
+SKILL_20151224_005859	Temporarily increase the fire, ice, lightning, poison and earth elemental resistance of you and your party members. At a certain rate, negate the listed elemental attack and decrease the magical defense of enemies nearby.
+SKILL_20151224_005860	Elemental resistance increase: #{CaptionRatio}#{nl}Elemental attack negation rate: #{CaptionRatio2}#%{nl}Enemy magical defense reduction: #{SkillFactor}#{nl}Duration: #{CaptionTime}#second
+SKILL_20151224_005861	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Maximum extinction count: #{CaptionRatio}#
+SKILL_20151224_005862	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Strike with powerful hand knife to damage the enemy.
+SKILL_20151224_005863	Maximum purchase count: #{CaptionRatio}#
+SKILL_20151224_005864	Decatose
+SKILL_20151224_005865	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Attack with the collected money in order to offer to the gods. Enemies, but boss can be instantly killed at 1/10 rate and the money from instant kill can be picked back up.
+SKILL_20151224_005866	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Silver consumption: #{CaptionRatio}#
+SKILL_20151224_005867	Duration: #{CaptionRatio}#second{nl}Master level 1
+SKILL_20151224_005868	{#993399}{ol}[Magical] - [Holy]{/}{/}{nl}Temporarily creates magic circle which grants hyper regeneration. If you or your comrades are damaged by an enemy attack inside of the magic circle, HP will be quickly recovered.
```

Comment:
[Magical] -> [Magic]

Temporarily creates a magic circle that grants you a healing factor. If you or your allies take damage from an enemy inside the magic circle, HP will be quickly recovered.

(I think healing factor is known more in the western world?)